libraries/AP_HAL_ChibiOS/hwdef/NxtPX4v2/README.md

```diff
@@ -0,0 +1,92 @@
+# NxtPX4v2 Flight Controller
+
+The NxtPX4v2 is an open-source hardware designed and maintened by [HKUST UAV-Group](https://github.com/HKUST-Aerial-Robotics/Nxt-FC). And it is produced by [MicoAir Tech.](http://micoair.com/).
```

Comment:
if open source should submit schematics,etc. to the ArduPilot/Schematics repo

Reply:
Got it. I have submitted the PCB files to the ArduPilot/Schematics repo. [#10](https://github.com/ArduPilot/Schematics/pull/10)
Please take a look.